edi/models/edi_exchange_consumer_mixin.py

```diff
@@ -19,6 +25,104 @@ class EDIExchangeConsumerMixin(models.AbstractModel):
         domain=lambda r: [("model", "=", r._name)],
     )
     exchange_record_count = fields.Integer(compute="_compute_exchange_record_count")
+    expected_edi_configuration = fields.Serialized(
+        compute="_compute_expected_edi_configuration"
+    )
+    has_expected_edi_configuration = fields.Boolean(
+        compute="_compute_expected_edi_configuration"
+    )
+
+    def _compute_expected_edi_configuration(self):
+        for record in self:
+            configurations = record._get_expected_edi_configuration()
+            record.expected_edi_configuration = configurations
+            record.has_expected_edi_configuration = bool(configurations)
+
+    def _get_expected_edi_configuration(self):
+        exchange_types = (
+            self.env["edi.exchange.type"]
+            .sudo()
+            .search([("model_ids.model", "=", self._name), ("backend_id", "!=", False)])
```

Comment:
why backend_id False? This excludes all types that are not assigned to a specific backend but only to a backend type.

Reply:
Because I need a backend for generating the exchange record and I only have the record and the exchange type :thinking:

Reply:
True, I will work on it :wink:

Reply:
Done